### PR TITLE
enhancement(partition-plan): add a prompt when no preview sql generated

### LINKED ISSUE #2345: [Bug]: The parentheses for database changes are in English
### ODC version

odc424

### OB version

*

### What happened?

image

### What did you expect to happen?

Brackets in Chinese

### How can we reproduce it (as minimally and precisely as possible)?

Execute database changes

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by yhilmare**
#### What type of PR is this?
type-enhancement

#### What this PR does / why we need it:
add some prompt when no preview sql generated
截屏2024-05-09 20 56 52
截屏2024-05-09 20 57 23

#### Which issue(s) this PR fixes:

Fixes #2345

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-core/src/main/java/com/oceanbase/odc/core/shared/constant/ErrorCodes.java [hunk 1]
```diff
@@ -164,6 +164,8 @@ public enum ErrorCodes implements ErrorCode {
     InvalidCronExpression,
 
     // Partition plan
+    NoPartitionDropPreviewSqlGenerated,
```
Comment (yizhouxw) [inline server/odc-core/src/main/java/com/oceanbase/odc/core/shared/constant/ErrorCodes.java:167]: PartitionPlanNoDropPreviewSqlGenerated
PartitionPlanNoCreatePreviewSqlGenerated\

start with module name

Comment (yhilmare) [reply server/odc-core/src/main/java/com/oceanbase/odc/core/shared/constant/ErrorCodes.java]: done

**Review by yizhouxw — APPROVED** [review]
lgtm

**Review by krihy — APPROVED** [review]
lgtm

**Pull request merged**